src/Microsoft.PowerShell.ConsoleHost/host/msh/ManagedEntrance.cs

```diff
@@ -106,12 +106,13 @@ public int Start(string consoleFilePath, string[] args)
                     RunspaceConfigForSingleShell.Create(consoleFilePath, out warning);
             }
             int exitCode = 0;
+            string systemYear = DateTime.Now.Year.ToString();
```

Comment:
Maybe I was not accurate. This variable is under the `#if` directive and actually used once in the result code.